Ref Guide - Channel List (#282)
uyuni-project/uyuni-docs: modules/reference/pages/software/software-menu.adoc

[[ref.webui.software]]
= Software Menu

> The [guimenu]``Software`` section allows you to view and manage software channels and packages on your clients.

Reviewer: ```suggestion
The [guimenu]``Software`` section allows you to view and manage software channels, repositories, and packages.
```
With these menus you manage channels on the server, IMO.  This, later on, helps with managing software on the clients.
